allennlp/models/bidaf.py

```diff
@@ -350,7 +329,7 @@ def from_params(cls, vocab: Vocabulary, params: Params) -> 'BidirectionalAttenti
         span_end_encoder = Seq2SeqEncoder.from_params(params.pop("span_end_encoder"))
         initializer = InitializerApplicator.from_params(params.pop("initializer", []))
         dropout = params.pop('dropout', 0.2)
-        evaluation_json_file = params.pop('evaluation_json_file', None)
+        params.pop('evaluation_json_file', None)  # Temporary, for backwards compatibility
```

Comment:
Maybe assign this to something and give a deprecation warning if it's not `None`?  And add a TODO to remove this once the saved models are updated?

Reply:
OK, do we have an existing pattern for deprecation warnings yet?

Reply:
No, we don't.